src/resources/discovery.js

```diff
@@ -36,7 +36,11 @@ class Discovery {
    * @param filters {object} object with properties: name, value, valueType, operator
    * @returns {Promise<HTTP_Response>}
    */
-  async search(searchQuery, filters = []) {
+  async search(searchQuery, itemsPerPage, fromPage, filters = []) {
+    // from page should be bigger than 0
+    fromPage = Math.max(fromPage, 1)
+    // clamp itemsPerPage between 1 and 12
```

Comment:
Should we give the client the flexibility to request more than that ? 12 seems small. I'd go for 100 ?

Reply:
yup agreed lets go for 100